### PR TITLE
[10.0][MIG] l10n_br_stock_account

### PR DESCRIPTION
**Pull request by mbcosta**
Replace https://github.com/OCA/l10n-brazil/pull/640 to use a common repository.

Depends https://github.com/OCA/account-fiscal-rule/pull/90

WIP - Procurement

cc @renatonlima @rvalyi @mileo

### CODE REVIEW

**Comment by rvalyi** [comment]
OCA/account-fiscal-rule#90 has been merged. This should be rebased to avoid the postgres error with .travis.yml.

**Comment by rvalyi** [comment]
I just did the rebase on 10.0, let's see the tests now...

**Comment by renatonlima** [comment]
I think the pot translation file is missing

File: l10n_br_stock_account/__manifest__.py [hunk 1]
```diff
@@ -8,26 +8,26 @@
...
     'author': 'Akretion, Odoo Community Association (OCA)',
     'website': 'http://odoo-brasil.org',
-    'version': '8.0.1.0.1',
+    'version': '10.0.1.0.1',
     'depends': [
         'l10n_br_account_product',
         'l10n_br_stock',
         'account_fiscal_position_rule_stock',
         'stock_account',
+        'sale_stock',
```
Comment (renatonlima) [inline l10n_br_stock_account/__manifest__.py]: Esta dependência deveria estar no módulo l10n_br_sale_stock. https://github.com/OCA/l10n-brazil/blob/10.0/l10n_br_sale_stock/__manifest__.py#L14

File: l10n_br_stock_account/views/stock_view.xml [hunk 2]
```diff
@@ -1,33 +1,27 @@
...
+            <field name="picking_type_id" position="after">
+                <field name="fiscal_category_id"/>
             </field>
-        </record>
-    </data>
-</openerp>
+        </field>
+    </record>
+
+    <!--<record model="ir.ui.view" id="l10n_br_stock_account_procurement_rule_inherit_form">
```
Comment (renatonlima) [inline l10n_br_stock_account/views/stock_view.xml:16]: Remover estes comentários

File: l10n_br_stock_account/wizard/stock_invoice_onshipping.py [hunk 3]
```diff
@@ -34,22 +31,152 @@ def open_invoice(self):
...
+                _('Invalid Journal!'),
+                _('There is not journal defined for this company: %s in '
+                  'fiscal operation: %s !') %
+                (picking.company_id.name,
+                 picking.fiscal_category_id.name))
+
+        comment = ''
+        if picking.fiscal_position_id.inv_copy_note:
+            comment += picking.fiscal_position_id.note or ''
+        if picking.sale_id and picking.sale_id.copy_note:
```
Comment (renatonlima) [inline l10n_br_stock_account/wizard/stock_invoice_onshipping.py]: Esta parte deste código deveria estar no módulo l10n_br_sale_stock

File: l10n_br_stock_account/demo/l10n_br_stock_account_demo.xml [hunk 4]
```diff
@@ -0,0 +1,86 @@
...
+        <field name="product_uom_qty">2</field>
+        <field name="product_uom" ref="product.product_uom_unit"/>
+        <field name="picking_id" ref="demo_l10n_br_stock_account-picking-1"/>
+        <field name="location_id" ref="stock.stock_location_stock"/>
+        <field name="location_dest_id" ref="stock.stock_location_customers"/>
+        <field name="invoice_state">2binvoiced</field>
+        <field name="fiscal_category_id" ref="l10n_br_account_product.fc_86d8c770fc2fb9d9fa242a3bdddd507a"/>
+    </record>
+
+    <!-- Picking Test - 2
```
Comment (renatonlima) [inline l10n_br_stock_account/demo/l10n_br_stock_account_demo.xml]: Remover comentários

**Review by renatonlima — CHANGES_REQUESTED** [review]

**Review by renatonlima — APPROVED** [review]

**Comment by renatonlima** [comment]
Kudos @mbcosta!

**Comment by mbcosta** [comment]
Thanks @renatonlima for review

**Review by gabrielcardoso21 — APPROVED** [review]

**Comment by mbcosta** [comment]
Hi @gabrielcardoso21 , thanks by review

**Pull request merged**